src/main/java/thePackmaster/cards/bellordpack/Midnight.java

```diff
@@ -0,0 +1,40 @@
+package thePackmaster.cards.bellordpack;
+
+import com.evacipated.cardcrawl.mod.stslib.cards.interfaces.OnObtainCard;
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.cards.curses.CurseOfTheBell;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.core.Settings;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import com.megacrit.cardcrawl.vfx.cardManip.ShowCardAndObtainEffect;
+import thePackmaster.cards.AbstractPackmasterCard;
+
+import static thePackmaster.SpireAnniversary5Mod.makeID;
+
+public class Midnight extends AbstractPackmasterCard implements OnObtainCard {
+    public final static String ID = makeID("Midnight");
+    // intellij stuff attack, enemy, common, 9, 3, 9, 3, , 
+
+    public Midnight() {
+        super(ID, 1, CardType.ATTACK, CardRarity.COMMON, CardTarget.ENEMY);
+        baseDamage = 9;
+        baseBlock = 9;
+        cardsToPreview = new CurseOfTheBell();
```

Comment:
Does it need to preview the card after it is already obtained? Not like it has any further effect on the card. Might be neat to remove it after pickup.